app/routes.js

```diff
@@ -17,7 +17,7 @@ router.get('/persons/all', PersonController.listAll);
 // Announcement Routes
 router.post('/convocatorias', AnnouncementController.save);
 
-router.get('/Convocatorias/list', AnnouncementController.findforfilter);
```

Comment:
y porque no utiliza el findforFilter?

Reply:
Profe  lo que pasa es que faltaba un listar todos y el findforfilter
 recibe unos parametros entonces por eso lo hize
 ¿profe, entonces el listar todos lo tengo que hacer por otra ruta o el find for filter ponerlo por otra ruta?

Reply:
listar todos en otra ruta porfa y deje el filtrar como estaba